# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfce4-terminal/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Terminal-0.2.4/terminal/terminal-shortcut-editor.c.orig	2006-02-13 19:37:31.000000000 +0100
+++ Terminal-0.2.4/terminal/terminal-shortcut-editor.c	2006-02-15 01:49:19.000000000 +0100
@@ -25,6 +25,7 @@
 #include <config.h>
 #endif
 
+#include <gdk/gdkx.h>
 #include <gdk/gdkkeysyms.h>
 
 #include <terminal/terminal-preferences.h>
--- Terminal-0.2.4/terminal/terminal-window.c.orig	2006-02-15 01:55:40.000000000 +0100
+++ Terminal-0.2.4/terminal/terminal-window.c	2006-02-15 01:56:01.000000000 +0100
@@ -39,6 +39,8 @@
 
 #include <exo/exo.h>
 
+#include <libxfcegui4/xfce_aboutdialog.h>
+
 #include <gdk/gdkkeysyms.h>
 #include <gdk/gdkx.h>
 
